src/PrestaShopBundle/Resources/views/Admin/layout.html.twig

```diff
@@ -56,35 +56,35 @@ app.request.attributes.get('_route')
             <button type="button" class="close" data-dismiss="alert" aria-label="Close">
                 <span aria-hidden="true"><i class="material-icons">close</i></span>
             </button>
-            <ul class="alert-text">
+            <div class="alert-text">
```

Comment:
I totally disagree with this. More this make the list of notifications unreadable for blind people readers :/ please consider a solution where we keep list when it's a list